marrow/mailer/transport/smtp.py

```diff
@@ -16,6 +17,7 @@
 
 log = __import__('logging').getLogger(__name__)
 
+PY35_OR_LATER = (sys.version_info >= (3,5))
```

Comment:
There seems to be no change attributed to Python 3.5 within the documentation:

* https://docs.python.org/3.5/library/smtplib.html#smtplib.SMTP  
  https://docs.python.org/3.8/library/smtplib.html?highlight=3.5#smtplib.SMTP

* https://docs.python.org/3.5/library/smtplib.html#smtplib.SMTP_SSL
  https://docs.python.org/3.8/library/smtplib.html?highlight=3.5#smtplib.SMTP_SSL

* https://docs.python.org/3.5/library/smtplib.html#smtplib.SMTP.connect
  https://docs.python.org/3.8/library/smtplib.html?highlight=3.5#smtplib.SMTP.connect

There _is_ a deprecation in how the acceptable certificate context should be properly configured, but this does not seem to relate?